### DISCUSSION TITLE
帮我看一下这个是什么问题

### DISCUSSION THREAD

**Discussion by admimare** [monsterxcn/nonebot-plugin-gspanel#54] [提问]
![IMG_0811](https://github.com/monsterxcn/nonebot-plugin-gspanel/assets/74863664/7dc7671e-7e57-4563-8352-b645c03a9077)
是不是资源丢失

**Comment by admimare** [comment]
py版本3.10 运行环境小米6
帮忙看看 感谢

**Comment by monsterxcn** [comment]
环境变量填的有问题吧，`resources_dir` 自己研究下，具体我也不清楚。感觉和 #53 很像啊。

[这里](https://github.com/monsterxcn/nonebot-plugin-gspanel/tree/main/data/gspanel) 的所有文件应该在那个目录下。

**Comment by admimare** [comment]
好的 感谢


> 在 2023年6月4日，18:09，Monst.x ***@***.***> 写道：
> 
> resources_dir
